test/ASTMerge/struct/Inputs/struct1.c

```diff
@@ -61,3 +61,65 @@ struct {
   Int i;
   float f;
 } x11;
+
+//Matches
+struct Unnamed {
+  union {
+    struct {
+      int i;
+    } S;
+    struct {
+      float i;
+    } R;
+  } U;
+} x12;
+
+//Matches
```

Comment:
Make comments full sentences with a period or remove them if they do not convey any additional information.